tests/token/erc20/ERC20BaseSuite.py

```diff
@@ -57,7 +59,9 @@ async def test_constructor_exceed_max_decimals(self, contract_factory):
                     bad_decimals,
                     *INIT_SUPPLY,
                     account.contract_address
-                ]),
+                ],
+                cairo_path=get_cairo_path()
```

Comment:
why is this needed only here?

Reply:
It is needed here and also in the `compile_starknet_files` call because we don't install the library for coverage (`skip_install` option in tox). We need to specify the path for imports in calls where the compiler is involved (`starknet.deploy` and `compile_starknet_files` don't get the CAIRO_PATH env variable, so it needs to be passed directly).

The reason why we need to skip the installation for coverage to work well is that there are not relative imports in cairo. For the coverage overriding VM to recognize imported files as the ones in the project (and not in python/lib), we need to specify that the cairo path is the src folder and not the project's root.